sdks/js/packages/core/react/components/avatar-upload/index.tsx

```diff
@@ -122,6 +122,7 @@ function CropModal({ onClose, imgSrc, onSave }: CropModalProps) {
               onChange={(_, percentCrop) => setCrop(percentCrop)}
               aspect={1}
               className={styles.reactCrop}
+              data-test-id="image-crop-preview"
```

Comment:
use `frontier-sdk` as prefix for all  `data-test-id`. we are following that convention in the SDK JS code.